Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs

```diff
@@ -371,6 +378,18 @@ void ApplyIntegretionLibraries (string socialPlatform, bool remove)
 
 				EditorGUI.EndDisabledGroup();
 				break;
+			case "gameCenter":
+#if UNITY_IOS
```

Comment:
@EugeneButusov, are you sure you need `#if` here? What if an user has Android active at the moment, but he creates an app for both platform?